understanding/21/pointer-cancellation.html

```diff
@@ -13,7 +13,9 @@ <h2>Intent</h2>
 			<h3>Up-Event activation or completion</h3>
 			<p>The most accessible way to incorporate pointer cancellation is to make activation occur on the up-event.</p>
       
-        <p>Up-event activation refers to the activation of a target when the pointer is released. In a touchscreen interaction, when the finger touches a target, the up-event activation only occurs when the finger is lifted while still being within the target boundary. Similarly in mouse interaction, the up-event occurs when the mouse button is released while the cursor is still within the boundary of the initial target set when the mouse button was pressed.</p>
+        <p>Up-event activation refers to the activation of a target when the pointer is released. In a touchscreen interaction, when the finger touches a target, the up-event activation only occurs when the finger is lifted while still being within the target boundary. Similarly in mouse interaction, the up-event occurs when the mouse button is released while the cursor is still within the boundary of the initial target set when the mouse button was pressed. Note however that the touchend event, if defined by the author, is fired at the original target element even when the pointer is released outside the target boundary.</p>
```

Comment:
The last sentence is a little convoluted, does it mean the same as this?
> Note that if the author defines a touchend event, that event is triggered on the original target element, even if the pointer is released outside the target boundary.

Reply:
Yes, your sentence is better, it means the same thing.